mautic/mautic
# [UI] Place UTM tags under accordion

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#200 
| Issue(s) addressed                     | Fixes #... 



## Description

Dependencies:
- #13940 

This PR:
- Creates a reusable accordion component in Twig
- Replaces all manual ways to display UTM fields and centralizes in one file (creates a reusable UTM tags fields component) to help us maintain them
- Then place UTM tags field under the accordion everywhere, reducing cognitive load caused by the amount of total fields on the screen when creating emails
Example (before):
![image](https://github.com/user-attachments/assets/fb36d056-b33a-4630-995b-c90d1758c1c3)

- Adds padding-inline utilities, which applies both padding left and right using only one class, replacing the use of separate pr- (padding-right) and pl- (padding-left) classes for situations where you need them together.




| Before                                 | After
| -------------------------------------- | ---
| ![image](https://github.com/user-attachments/assets/76211635-3567-4875-bfbd-920033b11fe4)                                        | ![image](https://github.com/user-attachments/assets/609a7757-974f-46a9-b20d-27077c7f3011)


When opened, works just fine:
![image](https://github.com/user-attachments/assets/9818f62e-6de2-42e0-909d-32823804e72e)


---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Dynamic content > Create new > Check for the UTM tags on the panel
3. Repeat the same for Emails, Mobile notifications, Web notifications and Focus items
4. See if they are displaying all fields when clicked, with proper alignment
